client/my-sites/customer-home/locations/upsells/index.jsx

```diff
@@ -10,41 +9,45 @@ import { connect } from 'react-redux';
  */
 import isSiteChecklistComplete from 'state/selectors/is-site-checklist-complete';
 import isUnlaunchedSite from 'state/selectors/is-unlaunched-site';
-import { getSiteOption } from 'state/sites/selectors';
 import StatsBanners from 'my-sites/stats/stats-banners';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 
-const Upsells = ( {
-	isChecklistComplete,
-	isEstablishedSite,
-	siteId,
-	siteIsUnlaunched,
-	siteSlug,
-} ) => {
+const cardComponents = {
+	'home-banner-legacy-stats-banners': StatsBanners,
```

Comment:
😆 stats banners is increasingly poorly named. I think this is fine for now, but we should eventually retire this string uid in favor of the broken out banners. For example:
`home-banner-upwork`
`home-banner-gsuite`
`home-banner-google-my-business`

Reply:
https://github.com/orgs/Automattic/projects/148#card-35581009